app/Exceptions/Handler.php

```
@@ -0,0 +1,41 @@
+        'password',
+        'password_confirmation',
+    ];
+
+    /**
+     * Register the exception handling callbacks for the application.
+     *
+     * @return void
+     */
+    public function register()
+    {
+        $this->reportable(function (Throwable $e) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$e'. (Unused Code Rules)

(UnusedFormalParameter)